manifest.json

```diff
@@ -1,17 +1,17 @@
 {
-    "spark_version": "2.2.1",
+    "spark_version": "2.3.2",
     "default_spark_dist": {
         "hadoop_version": "2.7",
-        "uri": "https://downloads.mesosphere.com/spark/assets/spark-2.2.1-4-SNAPSHOT-bin-2.7.tgz"
+        "uri": "https://downloads.mesosphere.com/spark/assets/spark-2.3.2-1-SNAPSHOT-bin-2.7.3.tgz"
```

Comment:
Jenkins upload job is failing if an object with the same key exists, this complicated SNAPSHOT replacement. Overall, I think we should still keep some sort of versioning to avoid failures when a snapshot is overridden with some broken/incompatible changes.

Reply:
If this is the case then do we need the word `SNAPSHOT` at all? We can go with a simpler versioning system like `2.3.2.1` or `2.3.2-1`, if we make it known the last digit represents a dev build. Or perhaps a proper thing to do is tag the file w/ a date-time